Cargo.toml

```diff
@@ -17,15 +17,15 @@ travis-ci = { repository = "eqrion/cbindgen" }
 [dependencies]
 clap = { version = "2", optional = true }
 log = "0.4"
-serde = { version = "1.0", default-features = false, features = ["derive"]}
+serde = { version = "1.0.103", default-features = false, features = ["derive"]}
```

Comment:
Can you elaborate? Why do we need this change?

Reply:
As-is, the dependencies are incorrectly specified.  If you do `cargo +nightly update -Zminimal-versions && cargo +nightly build`, the build will fail.  Worse, any crate that depends on cbindgen will also fail to build if it tries building with its minimal dependencies.